ext/pcaprub_c/pcaprub.c

```diff
@@ -1241,12 +1241,12 @@ rbpacket_data(VALUE self)
 }
 
 #if defined(WIN32)
-static VALUE
+static void *
```

Comment:
This is referenced below:

```c
  result = (VALUE)rb_thread_call_without_gvl(
      rbpcap_thread_wait_handle_blocking,
	  fno, RUBY_UBF_IO, 0);
```

But the definition for `rb_thread_call_without_gvl` is a return type of `void *` not `VALUE`:

https://github.com/ruby/ruby/blob/e3e1e53098f3214ef12849b3441791465ad7c0a5/thread.c#L1656